api/src/test/java/org/projectnessie/tools/catalog/migration/api/ITHadoopToNessieCatalogMigrator.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.catalog.migration.api;
+
+import java.util.Collections;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.SupportsNamespaces;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.projectnessie.tools.catalog.migration.api.test.HiveMetaStoreRunner;
+
+public class ITHadoopToNessieCatalogMigrator extends AbstractTestCatalogMigrator {
+
+  protected static final int NESSIE_PORT = Integer.getInteger("quarkus.http.test-port", 19121);
+
+  protected static String nessieUri = String.format("http://localhost:%d/api/v1", NESSIE_PORT);
+
+  @BeforeAll
+  protected static void setup() throws Exception {
+    HiveMetaStoreRunner.startMetastore();
+
+    catalog1 = createHadoopCatalog(warehouse1.toAbsolutePath().toString(), "hadoop");
+    catalog2 = createNessieCatalog(warehouse2.toAbsolutePath().toString(), nessieUri);
+
+    createNamespaces();
+  }
+
+  @AfterAll
+  protected static void tearDown() throws Exception {
+    dropNamespaces();
+    HiveMetaStoreRunner.stopMetastore();
+  }
+
+  // disable large table test for IT to save CI time. It will be executed only for UT.
```

Comment:
This is no viable option - especially since Nessie is the primarily targeted destination catalog